src/grelu/interpret/motifs.py

```diff
@@ -317,3 +317,19 @@ def compare_motifs(
     scan["foldChange"] = scan.alt / scan.ref
     scan = scan.sort_values("foldChange").reset_index(drop=True)
     return scan
+
+
+def calculate_ism_weight(row: pd.Series, ism: np.ndarray) -> float:
```

Comment:
Currently, we average the ISM values over the motif without accounting for whether the shape of the ISM signal (i.e. relative score of different nucleotides at the position) matches that of the motif. Wouldn't it be better if we convolved the ISM matrix with the motif matrix - e.g. using `np.convolve` or `scipy.signal.convolve`?